src/config_rev

```diff
@@ -22,7 +22,11 @@ Args = parser.parse_args()
 
 os.chdir(Args.directory)
 
-rev = 'UNKNOWN_REV'
+if 'VERILATOR_CUSTOM_REV' in os.environ:
+    rev = os.environ['VERILATOR_CUSTOM_REV']
```

Comment:
It looks like this is more commonly called VERSION so we might as well clean up the input name.  This ok?

```suggestion
# Override for OS distro source package builders
if 'VERILATOR_SRC_VERSION' in os.environ:
    rev = os.environ['VERILATOR_SRC_VERSION']
```

Reply:
Of course. Thanks for taking a look at this.